benchmark/pkg/comparer/schemes/kstest.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schemes
+
+import (
+	"fmt"
+	"math"
+
+	"k8s.io/perf-tests/benchmark/pkg/util"
+
+	"github.com/dgryski/go-onlinestats"
```

Comment:
I think if there's no LICENSE file, we can't use this. You can write to the author, to add apache2 to the repo.